modules/docs/arrow-docs/src/main/kotlin/DivideHelper.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.domain
```

Comment:
@1Jajen1 I didn't see this before merging. Could you please make it a nicer package?